### PR TITLE
Add pool-simulator-api app

### PR DESCRIPTION
**Pull request by ribeirojose**

### CODE REVIEW

File: apps/gyro-eclp-api/gyro_eclp_api/src/eclp_prec_implementation.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    tauBeta38 = (D2(tauBeta[0].raw), D2(tauBeta[1].raw))
+    derived = DerivedParams(
+        tauAlpha=(tauAlpha38[0], tauAlpha38[1]),
+        tauBeta=(tauBeta38[0], tauBeta38[1]),
+        u=D2(u.raw),
+        v=D2(v.raw),
+        w=D2(w.raw),
+        z=D2(z.raw),
+        dSq=D2(dSq.raw),
+        # dAlpha=D2(dAlpha.raw),
```
Comment (luizakp) [inline apps/gyro-eclp-api/gyro_eclp_api/src/eclp_prec_implementation.py:76]: Do you know what this is for? If not I think we can remove the comment

File: apps/gyro-eclp-api/gyro_eclp_api/test_api.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        "u": "0.790905599558369850905339125610308",
+        "v": "0.6076383033720348083193297868072476",
+        "w": "-0.05585063777148738251815296884188865",
+        "z": "0.03975485570515915653508200992108477",
+        "dSq": "1.000000000000000000825967344137306",
+    }
+
+    # The api return the values with more decimal points. Since the math package
+    # handle the conversion from string to the right amount of decimals, we don't need to do it here.
+    # For test purposes we will round the values to 18 decimals.
```
Comment (luizakp) [inline apps/gyro-eclp-api/gyro_eclp_api/test_api.py:38]: 👍

**Review by luizakp — APPROVED** [review]
Great work, guys!

File: apps/gyro-eclp-api/gyro_eclp_api/src/eclp_prec_implementation.py (hunk 1 above)
Comment (ribeirojose) [reply apps/gyro-eclp-api/gyro_eclp_api/src/eclp_prec_implementation.py:76]: This was also commented in their code 😅

**Pull request merged**